packages/shared/routes/dashboard/settings/views/security/AppLock.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="typescript">
+    import { Dropdown, Text } from 'shared/components'
+    import { localize } from 'shared/lib/i18n'
+    import { activeProfile, updateProfile } from 'shared/lib/profile'
+
+    function assignTimeoutOptionLabel(timeInMinutes) {
+        if (timeInMinutes >= 60) {
+            return localize('times.hour', { values: { time: timeInMinutes / 60 } })
+        }
+
+        return localize('times.minute', { values: { time: timeInMinutes } })
+    }
+
+    const lockScreenTimeoutOptions = [1, 5, 10, 30, 60].map((time) => ({
+        value: time,
+        label: assignTimeoutOptionLabel(time),
+    }))
+</script>
+
+<Text type="h4" classes="mb-3">{localize('views.settings.appLock.title')}</Text>
+<Text type="p" secondary classes="mb-5">{localize('views.settings.appLock.description')}</Text>
+<Dropdown
+    onSelect={(option) => {
+        updateProfile('settings.lockScreenTimeout', option.value)
+    }}
```

Comment:
Should we move this handler to the script block?

Reply:
done